plugins/processors/k8sdecorator/stores/utils.go

```diff
@@ -30,11 +32,12 @@ func createContainerKeyFromMetric(tags map[string]string) string {
 	return k8sutil.CreateContainerKey(namespace, podName, containerName)
 }
 
-const (
-	// kubeAllowedStringAlphaNums holds the characters allowed in replicaset names from as parent deployment
+var (
+	// deploymentUnallowedRegExp holds the characters allowed in replicaset names from as parent deployment
 	// https://github.com/kubernetes/apimachinery/blob/master/pkg/util/rand/rand.go#L83
-	kubeAllowedStringAlphaNums = "bcdfghjklmnpqrstvwxz2456789"
-	cronJobAllowedString       = "0123456789"
+	deploymentUnallowedRegExp = regexp.MustCompile(`[^b-hj-np-tv-xz-z2-24-9]+`)
```

Comment:
Also don't need to do a range between z and 2. 
```go
deploymentUnallowedRegExp = regexp.MustCompile(`[^b-hj-np-tv-xz24-9]+`)
```

Reply:
Nice catch. A little bad with RegEx.